chaos.py

```diff
@@ -0,0 +1,27 @@
+#!/bin/python3
+
+import sys
+
+def minimumBribes(q):
+    # Complete this function
+    bribes = len(q)

```

Comment:
I don't understand how you can keep track of at most 2 bribes per person with a single |bribes| variable.